pkg/api/auth_middleware.go

```diff
@@ -125,11 +130,18 @@ func checkSecurityRequirements(r *http.Request,
 					return nil, err
 				}
 				user, err = userFromOIDC(ctx, logger, authService, oidcSession, oidcConfig)
+			case "saml_auth":
+				samlSession, err := sessionStore.Get(r, SAMLAuthSessionName)
+				if err != nil {
+					return nil, err
+				}
+				user, err = userFromSAML(ctx, logger, authService, samlSession, cookieAuthConfig)
```

Comment:
If it's a static configuration why isn't being held by the authenticator instead of being passed around with every request?

Reply:
Followed the blue print of OIDC with `oidcConfig` (I know it's not a good reason), not sure which authenticator you are referring though?